x-pack/plugins/ml/public/application/model_management/model_actions.tsx

```diff
@@ -213,18 +217,23 @@ export function useModelActions({
           item.model_type === TRAINED_MODEL_TYPE.PYTORCH &&
           canStartStopTrainedModels &&
           !isLoading &&
-          item.stats?.deployment_stats?.state === DEPLOYMENT_STATE.STARTED,
+          !!item.stats?.deployment_stats?.some((v) => v.state === DEPLOYMENT_STATE.STARTED),
         onClick: async (item) => {
-          const threadingParams = await getUserInputThreadingParams(item.model_id, {
-            numOfAllocations: item.stats?.deployment_stats?.number_of_allocations!,
+          const deploymentToUpdate = item.deployment_ids[0];
```

Comment:
just want to double check that `[0]` here is definitely correct, will this list only have one item?